spec/registry.xml

```diff
@@ -135,7 +135,79 @@
         </description>
       </option>
     </formatSignature>
+  </function>
+
+  <function name="date">
+    <description>
+      Date formatting
+    </description>
+
+    <formatSignature>
+      <input validationRule="iso8601"/>
+      <option name="style" values="full long medium short">
+        <description>
+          The predefined date formatting style to use.
+        </description>
+      </option>
+      <option name="calendar" values="buddhist chinese coptic dangi ethioaa ethiopic gregory hebrew indian islamic islamic-umalqura islamic-tbla islamic-civil islamic-rgsa iso8601 japanese persian roc">
+        <description>
+          Calendar to use.
+        </description>
+      </option>
+      <option name="numberingSystem" values="arab arabext bali beng deva fullwide gujr guru hanidec khmr knda laoo latn limb mlym mong mymr orya tamldec telu thai tibt">
+        <description>
+          Numbering system to use.
+        </description>
```

Comment:
While the numbering system and calendar are useful configurations, I think they should come from the locale and not from the API options.

Reply:
The numbering system and the calendar come from the locale (almost all of the settings for date/time come from the locale by default). These are override options.